indexBasecodeANSWER1c.html

```diff
@@ -0,0 +1,157 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+		<title>Flights to Mars</title>
+    <link href="design2.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+      <br>
+      <div id="wrapper">
+            <div id="mainphoto">&nbsp;</div>  <!-- end mainphoto -->
+            <div id="nav">
+                  <table id='table1' width='900'>
+                  <tr>
+                  <td align='center' width='110' >

```

Comment:
` >` linters will solve some of the spacing problems in the syntax. here is one for Vscode editors and I believe there are other similar to them in google for different IDEs  [HTML lint](https://marketplace.visualstudio.com/items?itemName=mkaufman.HTMLHint)